lms/models/hubspot.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+
+from sqlalchemy import BigInteger
+from sqlalchemy.orm import Mapped, mapped_column
+
+from lms.db import Base
+
+
+class HubSpotCompany(Base):
+    """Raw companies data imported from HS."""
+
+    __tablename__ = "hubspot_company"
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    hs_object_id: Mapped[int] = mapped_column(BigInteger, unique=True)
```

Comment:
In the migration PR, [I suggested not using an acronym](https://github.com/hypothesis/lms/pull/6294#discussion_r1616697054), so I guess that will affect this property name as well.